src/bp/ui-studio/src/web/views/OneFlow/diagram/TriggerEditor/index.tsx

```diff
@@ -36,14 +35,15 @@ const EditGoalModal: FC<Props> = props => {
   const [conditions, setConditions] = useState<Condition[]>([])
   const [currentFlowCondition, setCurrentFlowCondition] = useState()
   const [currentCondition, setCurrentCondition] = useState<Condition>()
-
-  const [forceSave, setForceSave] = useState(false)
+  const [topicName, setTopicName] = useState('')
 
   useEffect(() => {
     if (props.node) {
       const { conditions } = props.node
+      const { currentFlow } = props.diagramEngine.flowBuilder.props
```

Comment:
That's not a good way to access that property. Please use directly `props.currentFlow`. Same thing for the save object. Connect the dispatch directly to TriggerEditor instead (switchFlowNode and updateFlowNode)